docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,89 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+
+.. warning::
+
+    Currently ``GlobusApp`` only supports scripting-based use cases.
+
+    Hosted applications (i.e. web servers) should see :ref:`globus_authorizers` instead.
+
+
+:class:`GlobusApp` is a high level construct designed to simplify the process of
+of authenticating for Globus service clients.
+
+A ``GlobusApp`` will always require a programmatic entity called a "client". Depending
+on the type of ``GlobusApp`` in use, the client will play different roles in the
+request flow.
+
+*   A :class:`UserApp` drives interaction starting with a browser based login
+    flow. With this type of ``GlobusApp``, newly created resources (collections,
+    flows, etc.) are owned by the human user who has logged in (and thus granted the
+    app authorization tokens). Similarly resource access is evaluated based on that
+    user's permissions.
+
+    A :class:`UserApp` should be used in cases where a script simplifies a process
+    for a user, but the user is ultimately responsible for the actions taken by the
+    script. The `Globus CLI <https://docs.globus.org/cli/>`_ is a good example of
+    one such use case.
```

Comment:
```suggestion
    A :class:`UserApp` is appropriate when a script or local application simplifies a process
    for a user, but the user is still ultimately responsible for the actions performed.
    The `Globus CLI <https://docs.globus.org/cli/>`_ is one such example.
```

Reply:
The content for this piece now reads (in a table):

> Commonly used to perform actions as a specific end user (e.g., the `Globus CLI <https://docs.globus.org/cli/>`_)

Reply:
I switched back to "appropriate for"; with the other wording shift, I do think that "appropriate" is more accurate and reads better than "commonly used to"